### DISCUSSION TITLE
What's the use of ref option for register method

### DISCUSSION THREAD

**Discussion by CodAffection** [react-hook-form/react-hook-form#10933] [Q&A]
I've looked into RHF documentation here : https://react-hook-form.com/docs/useform/register#options
`ref` description or code example seems incomplete, any one please describe.

**Comment by CodAffection** [comment]
@bluebill1049 @Moshyfawn 
Sorry, it's been a few months since I asked this question, no answer yet.
I couldn't see a single example demonstrating how to pass ref to the register method.

**Comment by Dragate** [comment]
I guess you could extract the ref from register and merge it with yours before passing it to the form field.

```jsx
import mergeRefs from "merge-refs"

// ...code

const { ref, ...rest } = register("name")
const myRef = useRef()

return <input {...rest} ref={mergeRefs(ref, myRef)} />
```

**Comment by CodAffection** [comment]
@Dragate Thanks for sharing your opinion. 

But the given code block does not use the register method option `ref`.
I also went through most of the documentation, but I couldn't see a single code block showing the relevance of this option.
It seems like it was added in the documentation by mistake because I could see the option inside the library source code as well.
